File: Topic Based Problems/01 Fundamentals/1 Beginner/09 Celsius to Fahrenheit/Java/CelsiustoFahrenheit.java
import java.util.Scanner;

public class CelsiustoFahrenheit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        double celsius;

        System.out.print("Enter temperature in Celsius: ");
        celsius = scanner.nextDouble();

        System.out.printf("Temperature in Fahrenheit is: %.2f\n", (celsius * 9 / 5) + 32);

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/09 Celsius to Fahrenheit/C/CelsiustoFahrenheit.c
#include <stdio.h>

int main() {
    double celsius;

    printf("Enter temperature in Celsius: ");
    scanf("%lf", &celsius);

    printf("Temperature in Fahrenheit is: %.2lf\n", (celsius * 9 / 5) + 32);

    return 0;
}
